tutorials/04_physics_engines.md

```diff
@@ -1,3 +1,3 @@
 \page physicsengine Use different physics engines
 
-See [Physics engines tutorial](https://gazebosim.org/api/gazebo/3.8/physics.html) in Gazebo.
+See [Physics engines tutorial](https://gazebosim.org/api/gazebo/6/physics.html) in Gazebo.
```

Comment:
```suggestion
See [Physics engines tutorial](https://gazebosim.org/api/physics/7/physicsplugin.html) in Gazebo.
```

Reply:
hmm, actually, this should link to https://gazebosim.org/api/sim/8/physics.html since it's talking about the tutorial "in Gazebo" (gz-sim)

Reply:
Also, this was fixed in #684 so I'll backport that instead.